src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -212,11 +216,11 @@ public static class PublishedContentExtensions
                 if (cacheAttr != null)
                 {
                     var ctx = new DittoCacheContext(cacheAttr, content, type, culture);
-                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted));
+                    return cacheAttr.GetCacheItem(ctx, () => ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted));
                 }
                 else
                 {
-                    return ConvertContent(content, type, culture, instance, processorContexts, onConverting, onConverted);
+                    return ConvertContent(content, type, umbracoApplicationContextAccessor, culture, instance, processorContexts, onConverting, onConverted);
```

Comment:
Is there a reason why the `umbracoApplicationContextAccessor` is being passed in as a method parameter from this high up the chain?  I'm assuming it's to do with the lazy-loaded properties, but let's review 👍